awesome_owl/static/src/ToDo/todo_list.js

```diff
@@ -0,0 +1,38 @@
+/** @odoo-module **/
+
+import { Component, useState, setState } from "@odoo/owl";
+import { TodoItem } from "./todo_item"
+import { useAutofocus } from "../utils";
+
+export class TodoList extends Component {
+    static template = "awesome_owl.todo_list";
+
+    static components = { TodoItem };
+
+    setup() {
+        this.todos = useState(new Array());
+        useAutofocus("input");
+    }
+
+    onInputEnter(event) {
+        if (event.keyCode === 13 && event.target.value != "") {
```

Comment:
nitpick: more readable
```suggestion
        if (event.key === "Enter" && event.target.value) {
```
(I know the tutorials says but :woman_shrugging: )